develop/devguide/GQI_Extensions/Ad_hoc_data/Ad_hoc_Tutorials_Interact_With_DMS.md

```diff
@@ -0,0 +1,221 @@
+---
+uid: Ad_hoc_Tutorials_Interact_With_DMS
+---
+
+# Building a GQI data source that retrieves data from a DMS
+
+In this tutorial, you will learn how you can create a GQI data source that retrieves the version info of the Agents in your DMS.
+
+Expected duration: 15 minutes.
+
+> [!TIP]
+> See also: [Kata #35: Interact with your DMS using an ad hoc data source](https://community.dataminer.services/courses/kata-35/) on DataMiner Dojo ![Video](~/dataminer/images/video_Duo.png)
```

Comment:
If the Kata video is now outdated, perhaps we shouldn't include this link anymore, since it won't really be helpful for the user now. What do you think?

Reply:
I would tend to agree with removing it indeed. We shoudln't remove the kata but we could promote it less by removing such references to it. I believe the tuto is self-explanatory enough to not need to forward readers to the Kata.